### PR TITLE
Adds the Protein Bar Belt and Pouch

### PR DESCRIPTION
**Pull request by efzapa**
## About The Pull Request

Adds the Protein Belt and Pouch, which can hold 20 and 10 protein bars respectively.

## Why It's Good For The Game

Mostly a gimmick item since nutriment canteens exist already, but with new blood rework i thought it useful to have atleast SOME option to hold food en masse god forbid you wanted to run that.

## Changelog

:cl:
add: Adds the Protein Bar Belt and Pouch.
/:cl:

### CODE REVIEW

**Comment by Derrick2Electricboogaloo** [comment]
I approve.

**Review by QualityVan — COMMENTED** [review]
I'm not strictly opposed to this, but it does seem weird when MREs exist as a 'I want to spend more than one tiny slot on food' option.
Also not sure what's up with the max_w and max_storage_space given that the only thing it can hold is tiny.

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -1204,6 +1204,9 @@
 			/obj/item/weapon/twohanded/glaive/harvester = -1,
 			/obj/item/storage/belt/utility/full =-1,
 			/obj/item/storage/belt/medical_small = -1,
+			/obj/item/storage/belt/protein_pack = -1
+
+
```
Comment (ivanmixo) [inline code/game/objects/machinery/vending/marine_vending.dm:1209]:
```suggestion
```

**Review by ivanmixo — APPROVED** [review]

**Comment by efzapa** [comment]
> I'm not strictly opposed to this, but it does seem weird when MREs exist as a 'I want to spend more than one tiny slot on food' option. Also not sure what's up with the max_w and max_storage_space given that the only thing it can hold is tiny.

well after some testing, apparently without the `max_storage_space` var, it only lets you store 14 bars in a belt for some reason, so you do need that.

you are right about the size req var however which i changed

**Comment by QualityVan** [comment]
Did you set the max storage size to 20 when you tried? If not, it'll inherit storage's default max_storage_size which's 14.

**Comment by efzapa** [comment]
I'll fix merge tonight since dmi conflict are an ass

**Comment by efzapa** [comment]
well i give up

**Pull request closed without merging**